__tests__/integration.test.js

```diff
@@ -354,3 +354,46 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  it("DELETE: 204 deletes the comment by given comment ID.", () => {
+    return request(app).delete("/api/comments/1").expect(204);
+  });
+
+  it("DELETE: 400 should handle invalid comment ID.", () => {
+    return request(app)
+      .delete("/api/comments/not")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request.");
+      });
+  });
+
+  it("DELETE: 404 sends an appropriate status and error message when given a valid but non-existent id.", () => {
+    return request(app)
+      .delete("/api/comments/1000")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Comment not found.");
+      });
+  });
+});
```

Comment:
I think this is ticket 9 - it shouldnt be in this PR. You'd have a horrible time if you needed to make changes as you've then got code that needs to be changed in two places potentially, which isnt how branches are meant to work.